userspace/libsinsp/examples/test.cpp

```diff
@@ -194,7 +194,7 @@ void open_engine(sinsp& inspector)
 	}
 	else if(!engine_string.compare(MODERN_BPF_ENGINE))
 	{
-		inspector.open_modern_bpf(buffer_bytes_dim, ppm_sc, tp_set);
+		inspector.open_modern_bpf(buffer_bytes_dim, DEFAULT_CPU_FOR_EACH_BUFFER, true, ppm_sc, tp_set);
```

Comment:
If I see this right, in `scap-open` the default is `false` where for `libsinsp` it is `true`. I like just to confirm this difference.

Reply:
Yeah IMHO the default should be only `online` CPUs just to be compliant with the other 2 drivers, but this is a good point I can change it in the scap-open thank you :)

Reply:
You are welcome. Thank you!